src/django/api/models/reference_image.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from .boundary import Boundary
+from .user import User
+
+__all__ = ["ReferenceImage"]
+
+
+class ReferenceImage(models.Model):
+    filename = models.CharField(max_length=100, blank=True)
+    boundary = models.ForeignKey(Boundary, on_delete=models.CASCADE)
```

Comment:
For `on_delete`, prefer `models.PROTECT` in most cases, except those where accidental deletion of related data is tolerable. I think in this app, we should use `models.PROTECT` in all cases.